### ISSUE TITLE
Build Data File

### ISSUE THREAD

**Issue by JonathanPMohan** [JonathanPMohan/booze-clues#3]
## User Story

As a developer, I will need seed data to build my project.

## Acceptance Criteria

• Create seed data for locations, collection and clues

}
    "clue0" : {
    "imageurl" : "",
    "name" : "",
    "style" : "",
    "location" : "",
    "notes" : "",
    }

Stretch Goals: Locations
    {
    "location0" : {
    "imageurl" : "",
    "name" : "",
    "address" : "",
    "phoneNumber" : "",
    "website" : "",
    }
Stretch Goals:  Personal Collection
   }
    "item0" : {
    "imageurl" : "",
    "style" : "",
    "ageVintage" : "",
    "notes" : "",
    }
    
## Tech Notes

• Once seed data is complete, you may move on.
